psalm-report.json

```diff
@@ -1,5 +1,644 @@
+            "selected_text": "$adyenApiRequestTransfer",
+            "error_level": 7,
+            "shortcode": 230,
+            "other_references": null
+        },
+        {
+            "severity": "error",
+            "line_from": 19,
+            "line_to": 19,
+            "type": "ParamNameMismatch",
+            "message": "Argument 1 of SprykerEco\\Zed\\AdyenApi\\Business\\Mapper\\PaymentDetailsMapper::buildRequestArray has wrong name $adyenApiRequestTransfer, expecting $requestTransfer as defined by SprykerEco\\Zed\\AdyenApi\\Business\\Mapper\\AdyenApiMapperInterface::buildRequestArray",
+            "file_name": "vendor/spryker-eco/adyen-api/src/SprykerEco/Zed/AdyenApi/Business/Mapper/PaymentDetailsMapper.php",
+            "snippet": "    public function buildRequestArray(AdyenApiRequestTransfer $adyenApiRequestTransfer): array",
+            "selected_text": "$adyenApiRequestTransfer",
+            "error_level": 7,
+            "shortcode": 230,
+            "other_references": null
+        },
+        {
+            "severity": "error",
+            "line_from": 19,
+            "line_to": 19,
+            "type": "ParamNameMismatch",
+            "message": "Argument 1 of SprykerEco\\Zed\\AdyenApi\\Business\\Mapper\\RefundMapper::buildRequestArray has wrong name $adyenApiRequestTransfer, expecting $requestTransfer as defined by SprykerEco\\Zed\\AdyenApi\\Business\\Mapper\\AdyenApiMapperInterface::buildRequestArray",
+            "file_name": "vendor/spryker-eco/adyen-api/src/SprykerEco/Zed/AdyenApi/Business/Mapper/RefundMapper.php",
+            "snippet": "    public function buildRequestArray(AdyenApiRequestTransfer $adyenApiRequestTransfer): array",
+            "selected_text": "$adyenApiRequestTransfer",
+            "error_level": 7,
+            "shortcode": 230,
+            "other_references": null
+        },
+        {
+            "severity": "error",
+            "line_from": 19,
+            "line_to": 19,
+            "type": "ParamNameMismatch",
+            "message": "Argument 1 of SprykerEco\\Zed\\AdyenApi\\Business\\Mapper\\TechnicalCancelMapper::buildRequestArray has wrong name $adyenApiRequestTransfer, expecting $requestTransfer as defined by SprykerEco\\Zed\\AdyenApi\\Business\\Mapper\\AdyenApiMapperInterface::buildRequestArray",
+            "file_name": "vendor/spryker-eco/adyen-api/src/SprykerEco/Zed/AdyenApi/Business/Mapper/TechnicalCancelMapper.php",
+            "snippet": "    public function buildRequestArray(AdyenApiRequestTransfer $adyenApiRequestTransfer): array",
+            "selected_text": "$adyenApiRequestTransfer",
+            "error_level": 7,
+            "shortcode": 230,
+            "other_references": null
+        }
+    ],
+    "warning": [
+        {
+            "severity": "info",
+            "line_from": 85,
+            "line_to": 85,
+            "type": "PossiblyUndefinedVariable",
+            "message": "Possibly undefined variable $options, first seen on line 85",
```

Comment:
Those seem to be quite valid issues that might need fixing :)

Reply:
Fixed